antarest/study/storage/variantstudy/model/command/create_link.py

```diff
@@ -43,11 +44,31 @@ class AbstractLinkCommand(ICommand, metaclass=ABCMeta):
 
     area1: str
     area2: str
-    parameters: Optional[Dict[str, Any]] = None
+    parameters: LinkUpdate
     series: Optional[Union[List[List[MatrixData]], str]] = None
     direct: Optional[Union[List[List[MatrixData]], str]] = None
     indirect: Optional[Union[List[List[MatrixData]], str]] = None
 
+    # version 2: replace type dict[str, Any] by class LinkUpdate for `parameters`
+    _SERIALIZATION_VERSION: Final[int] = 2
+
+    @model_validator(mode="before")
+    @classmethod
+    def _validate_parameters(cls, values: Dict[str, Any], info: ValidationInfo) -> Dict[str, Any]:
+        if "parameters" not in values:
+            values["parameters"] = LinkUpdate()
+
+        elif isinstance(values["parameters"], dict):
+            parameters = values["parameters"]
+            if info.context:
+                version = info.context.version
+                if version < 2:
+                    parameters.pop("area1", None)
```

Comment:
Actually I think we don't need to remove area1 and area2: those fields were only introduced temporarily with the DAO merge, they are not present in releases